src/components/standalone/dashboard/ServiceCard.vue

```diff
@@ -0,0 +1,171 @@
+<!--
+  Copyright (C) 2023 Nethesis S.r.l.
+  SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+<script setup lang="ts">
+import { ubusCall } from '@/lib/standalone/ubus'
+import { NeBadge, NeSkeleton, getAxiosErrorMessage } from '@nethserver/vue-tailwind-lib'
+import { onMounted, onUnmounted, ref, type Ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import NeCard from '@/components/NeCard.vue'
+
+const props = defineProps({
+  serviceName: { type: String },
+  hasStatus: { type: Boolean, default: false },
+  hasCounter: { type: Boolean, default: false },
+  title: {
+    type: String
+  },
+  icon: {
+    type: Array<string>
+  }
+})
+
+const { t } = useI18n()
+const STATUS_REFRESH_INTERVAL = 10000
+const COUNTER_REFRESH_INTERVAL = 10000
+const serviceStatus = ref<any>(null)
+const serviceCounter = ref<any>(null)
+const statusIntervalId: Ref<number> = ref(0)
+const counterIntervalId: Ref<number> = ref(0)
+
+let loading = ref({
+  getServiceStatus: false,
+  getServiceCounter: false
+})
+
+let error = ref({
+  title: '',
+  description: ''
+})
+
+onMounted(() => {
+  if (props.hasStatus) {
+    getServiceStatus()
+
+    // periodically reload data
+    statusIntervalId.value = setInterval(getServiceStatus, STATUS_REFRESH_INTERVAL)
+  }
+
+  if (props.hasCounter) {
+    getServiceCounter()
+
+    // periodically reload data
+    counterIntervalId.value = setInterval(getServiceCounter, COUNTER_REFRESH_INTERVAL)
+  }
+})
+
+onUnmounted(() => {
+  if (statusIntervalId.value) {
+    clearInterval(statusIntervalId.value)
+  }
+
+  if (counterIntervalId.value) {
+    clearInterval(counterIntervalId.value)
+  }
+})
+
+async function getServiceStatus() {
+  // show skeleton only the first time
+  if (!statusIntervalId.value) {
+    loading.value.getServiceStatus = true
```

Comment:
same as RealTimeTrafficChart, this could be set true by default

Reply:
In this case this doesn't work, since some cards need to retrieve just the status, while others need to retrieve just a counter, and they will never set `loading.getServiceStatus` or `loading.getServiceCounter' to `false`